components/comments-content.js

```diff
@@ -0,0 +1,48 @@
+import PropTypes from 'prop-types'
+import {Pane, Text} from 'evergreen-ui'
+
+const COMMENTS_LIMIT = 10
+
+function CommentsList({comments}) {
+  return comments.map(({_id, numero, suffixe, comment}) => (
+    <Pane color='white' key={_id} whiteSpace='nowrap' overflow='hidden' textOverflow='ellipsis'>
+      <Text color='white'>
+        {numero}{suffixe} : {comment}
+      </Text>
+    </Pane>
+  ))
+}
+
+function CommentsContent({comments}) {
+  const filteredComments = comments.length > COMMENTS_LIMIT ? comments.slice(0, COMMENTS_LIMIT) : comments
+  const nbComments = comments.length
+  const remainComments = nbComments - COMMENTS_LIMIT
+
+  return (
+    <>
+      <Pane marginBottom={8}>
+        <Text color='white'>
+          Commentaire{`${filteredComments.length > 0 ? 's' : ''}`}
```

Comment:
Ça ne change pas grand-chose, on est d'accord, mais c'est logique.
```suggestion
          Commentaire{`${comments.length > 0 ? 's' : ''}`}
```